src/components/Form/step_5.tsx

```diff
@@ -20,7 +20,7 @@ const Step5: React.FC<StepProps> = ({setStepCompleted}: StepProps) => {
  };
 
  return (
-  <div className=" py-4 md:py-6 lg:py-9 lg:h-[80vh] flex flex-col gap-14 md:gap-20 lg:gap-10 xl:gap-20 justify-between items-between">
+  <div className=" py-4 md:py-6 lg:py-9 lg:max-h-[80vh] flex flex-col gap-14 md:gap-20 lg:gap-10 xl:gap-20 justify-between items-between">
```

Comment:
![localhost-3001-booking-5](https://github.com/user-attachments/assets/9ba28028-f61a-4426-bcb0-41023eddadeb)
It should be like in the previous steppes lg:min-h-[80vh]